src/Servers/Kestrel/Core/src/HttpProtocols.cs

```diff
@@ -12,6 +12,7 @@ public enum HttpProtocols
         Http1 = 0x1,
         Http2 = 0x2,
         Http1AndHttp2 = Http1 | Http2,
+        // TODO Should we add Http1AndHttp2AndHttp3? Probably not.
```

Comment:
We have to add this 😄

Reply:
Honestly I prefer to break this behavior or something.

Reply:
Http1AndHttp2AndHttp3 would be amazingly convenient for our users, especially if it's the default.

Reply:
But the name...